src/Raven.Server/Documents/Handlers/StatsHandler.cs

```diff
@@ -56,5 +57,12 @@ public async Task BytesMetrics()
             using (var processor = new StatsHandlerProcessorForGetMetricsBytes(this))
                 await processor.ExecuteAsync();
         }
+
+        [RavenAction("/databases/*/validate-unused-ids", "GET", AuthorizationStatus.ValidUser, EndpointType.Read)]
```

Comment:
```
[RavenAction("/admin/databases/unused-ids", "POST", AuthorizationStatus.Operator)]
```
and this one can be executed by any user? why? also why in this handler?

Reply:
Changed the `AuthorizationStatus`
@karmeli87 thought it was right to put it in this handler because it is doing something similar to bringing stats ..

Reply:
Changed it to:
`[RavenAction("/databases/*/admin/validate-unused-ids", "POST", AuthorizationStatus.DatabaseAdmin)]`
it cannot be `Operator` according to the Authentication tests.